test/dotnet-publish.Tests/PublishAppWithDependencies.cs

```diff
@@ -0,0 +1,46 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.IO;
+using Microsoft.DotNet.TestFramework;
+using Microsoft.DotNet.Tools.Test.Utilities;
+using Xunit;
+
+namespace Microsoft.DotNet.Tools.Publish.Tests
+{
+    public class PublishAppWithDependencies : TestBase
+    {
+        [Fact]
+        public void PublishTestAppWithContentPackage()
```

Comment:
Which files shouldn't be there?  The other ones in that NuGet package that aren't CopyToOutput?